app/views/layouts/application.html.erb

```diff
@@ -25,6 +25,7 @@
   <% end %>
   <meta name="govuk:content-id" content="<%= @content_item.content_id %>" />
   <%= guide_test_variant.analytics_meta_tag.html_safe if is_tested_guide? %>
+  <%= contextual_comms_test_variant.analytics_meta_tag.html_safe %>
```

Comment:
This is the other thing where we should perhaps only include if it's relevant.  This is the thing that sends the analytics